### ISSUE TITLE
copyright

### ISSUE THREAD

**Issue by pirate486743186** [HDoujinDownloader/HDoujinDownloader#24]
You should add a copyright notice.

Also, you scrape random websites with a proprietary application and making money from it.
Some of that stuff are fan art, others are copyright infringement.
You risk having your ass handed to you.
youtube-dl for example is fully open source and in the public domain and avoids pirate sites. 

And people don't like contributing to proprietary applications, this is why they are zero PRs.

~~This seams to be a pandemic project, so maybe you haven't fully thought it threw yet.~~
EDIT: I saw 2015 somewhere

**Comment by HDoujinDownloader** [comment]
HDoujin Downloader is, and always has been, fully compliant with requests from rights holders (Fakku is a prominent example). The program's purpose is to download art and doujinshi material, which is most often made available to the public by the artists themselves.

If you feel users are scraping your content and they shouldn't be, just let me know!

**Comment by pirate486743186** [comment]
lol, just before answering i noticed you edited your reply.

There was  a proprietary program, in which one of the variables was "fucked". You shouldn't be too shy about your coding.

Then maybe you should announce something. And pick some license scheme. And what will be your goals? What about the people that bought it already? You'll be making less money from donations. Picking a license can be quite tricky and is essentially irreversible because then all contributors need to agree unanimously.... Or have copyright assignment, but people don't like that.

And there's open source and "open source". Usually the term "open source" means free software, they are some proprietary software with ... "available source". Given you haven't pick a license, your modules are in the second category. They aren't "really" open source.

**Comment by HDoujinDownloader** [comment]
Haha, well, maybe so. In the years I've spent maintaining this program I've learned a lot, but in the process accumulated a ton of spaghetti that even I can barely make sense of sometimes. I still take a _little bit_ of pride in my work, so I really want to clean it up as much as I can first. Right now what I'm doing is moving large swaths of it into separate projects that I'll be releasing separately as libraries. They will be released under an MIT license.

A genuine thanks for your input, dude. You bring up some very good points. As far as these modules go, it would save a lot of headache to establish a licensing scheme sooner rather than later. I'll give it some thought. 👍
